tests/framework/microvm.py

```diff
@@ -345,9 +347,6 @@ def pid_in_new_ns(self):
 
         Reads the pid from a file created by jailer with `--new-pid-ns` flag.
         """
-        # Check if the pid file exists.
-        assert self.jailer.pid_file.exists()
```

Comment:
if the pid file doesn't exist, the `read_text` later is going to fail anyway, isn't it?

Reply:
yeah it will, and it will raise `FileNotFoundError`, which we catch above. I though it better than catching `AssertionError`, which is less explicit.